app/src/main/java/com/islaharper/dawnofandroid/presentation/components/InfoBox.kt

```diff
@@ -0,0 +1,83 @@
+package com.islaharper.dawnofandroid.presentation.components
+
+import android.content.res.Configuration.UI_MODE_NIGHT_YES
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.ContentAlpha
+import androidx.compose.material.Surface
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import com.islaharper.dawnofandroid.R
+import com.islaharper.dawnofandroid.ui.theme.DawnOfAndroidTheme
+import com.islaharper.dawnofandroid.ui.theme.INFO_ICON_SIZE
+import com.islaharper.dawnofandroid.ui.theme.PADDING_SMALL
+
+@Composable
+fun InfoBox(
+    icon: Painter,
+    iconColour: Color,
+    mainText: String,
+    description: String,
+    textColour: Color,
+    modifier: Modifier = Modifier
+) {
+    Row(
+        modifier = Modifier
+            .semantics(mergeDescendants = true) {},
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Icon(
+            painter = icon,
+            contentDescription = null, // decorative
+            modifier = modifier
+                .padding(end = PADDING_SMALL)
+                .size(INFO_ICON_SIZE),
```

Comment:
Nit, I wouldn't extract a constant in a separate file if it's only going to be used in one place. It creates separation and developers have to flip back and forth to understand the code.

Better to inline it (Ctrl Alt + N) or to put it as `private` in the same file.

Reply:
Refactored and committed change.